docs/application/dotnet/guides/text-input/input-method.md

```diff
@@ -86,6 +86,41 @@ To start the IME application life-cycle:
     }
     ```
 
+## Making a Floating Keyboard
```

Comment:
Suggestion: 
Can we change the heading as : Use Floating Keyboard
As per my understanding we don't make/create floating keyboard.